client/src/components/DataFiles/DataFilesModals/DataFilesDownloadMessageModal.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Button, Modal, ModalHeader, ModalBody, ModalFooter } from 'reactstrap';
+import './DataFilesDownloadMessageModal.module.scss';
+
+const DataFilesDownloadMessageModal = () => {
+  const isOpen = useSelector(state => state.files.modals.downloadMessage);
+
+  const dispatch = useDispatch();
+  const toggle = () => {
+    dispatch({
+      type: 'DATA_FILES_TOGGLE_MODAL',
+      payload: { operation: 'downloadMessage', props: {} }
+    });
+  };
+
+  return (
+    <Modal isOpen={isOpen} toggle={toggle} size="md" className="dataFilesModal">
+      <ModalHeader toggle={toggle}>Download</ModalHeader>
+      <ModalBody>
+        <p styleName="title">Folders must be compressed before download.</p>
+        <ol>
+          <li>Select the folder(s).</li>
+          <li>Press the &quot;↩ Compress&quot; icon above the table.</li>
```

Comment:
The actual Compress icon should be used. This is possible.

<details>
<summary>How</summary>

```suggestion
          <li>
            Press the &quot;<Icon name="compress">↩</Icon> Compress&quot;
            icon above the table.
          </li>
```

> __Note__: You will need to import `Icon` from `_common`.

</details>

<details>
<summary>Screenshot</summary>

![Use Compress Icon](https://user-images.githubusercontent.com/62723358/122459231-4a071200-cf76-11eb-8cbf-ba594b4b9d87.png)

</details>